archetypes/Pools/FilterSelects/Bar.tsx

```diff
@@ -41,143 +43,175 @@ const REBALANCE_OPTIONS_MOBILE = [
-                        color={'tracer'}
-                        options={REBALANCE_OPTIONS_DESKTOP}
-                    />
-                </div>
-                <div className="flex-grow" />
-                <div className="hidden lg:flex mx-4 flex-col">
-                    <span className="inline-flex">
-                        <ArrowDown className="rotate-180 text-green-600 h-6" />
-                        <ArrowDown className="text-red-600 mr-1 h-6" />
-                        <h3 className="mb-1 text-theme-text">Denotion</h3>
-                    </span>
-                    <div className="mt-auto ml-auto">
-                        <TWButtonGroup
-                            value={state.deltaDenotion}
-                            onClick={(option) => dispatch({ type: 'setDenotion', denotion: option as DeltaEnum })}
-                            color="greyed"
-                            border="rounded"
-                            borderColor="greyed"
-                            options={DENOTION_OPTIONS}
+            <div className="w-full mb-2">
+                <div className="lg:flex">
+                    <div className="hidden lg:flex mr-4 flex-col">
+                        <h3 className="mb-1 text-theme-text">Market</h3>
+                        <Dropdown
+                            variant="blue"
+                            iconSize="xs"
+                            placeHolderIcon={
+                                Object.entries(MarketFilterEnum).find(
+                                    ([_key, val]) => val === state.marketFilter,
+                                )?.[0] as LogoTicker
+                            }
+                            value={state.marketFilter}
+                            className="w-48 mt-auto"
+                            options={Object.keys(MarketFilterEnum).map((key) => ({
+                                key: (MarketFilterEnum as any)[key],
+                                ticker: (key !== 'All' ? key : '') as LogoTicker,
+                            }))}
+                            onSelect={(val) => dispatch({ type: 'setMarketFilter', market: val as MarketFilterEnum })}
+                        />
+                    </div>
+                    <div className="hidden lg:flex mr-4 flex-col">
+                        <h3 className="mb-1 text-theme-text">Power Leverage</h3>
+                        <Dropdown
+                            value={state.leverageFilter}
+                            className="w-32 mt-auto"
+                            options={Object.keys(LeverageEnum).map((key) => ({
+                                key: (LeverageEnum as any)[key],
+                                ticker: (key !== 'All' ? key : '') as LogoTicker,
+                            }))}
+                            onSelect={(val) => dispatch({ type: 'setLeverageFilter', leverage: val as LeverageEnum })}
```

Comment:
can remove the icon on these 
![image](https://user-images.githubusercontent.com/29347276/153515876-4809cc59-056b-4350-b6c0-1626c853d776.png)

Reply:
✅